server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -608,13 +644,28 @@ public void deleteStorageGroups(List<PartialPath> storageGroups) throws Metadata
           logWriter.deleteStorageGroup(storageGroup.getFullPath());
         }
       }
-    } catch (ConfigAdjusterException e) {
-      throw new MetadataException(e);
     } catch (IOException e) {
       throw new MetadataException(e.getMessage());
     }
   }
 
+  /**
+   * update statistics in schemaDataTypeNumMap
+   *
+   * @param type data type
+   * @param num 1 for creating timeseries and -1 for deleting timeseries
+   */
+  private void updateSchemaDataTypeNumMap(TSDataType type, int num) {
+    schemaDataTypeNumMap.put(type, schemaDataTypeNumMap.getOrDefault(type, 0) + num);
+    schemaDataTypeNumMap.put(TSDataType.INT64,
+        schemaDataTypeNumMap.getOrDefault(TSDataType.INT64, 0) + num);
+    int currentDataTypeTotalNum = schemaDataTypeNumMap.values().size();
+    if (num > 0 && currentDataTypeTotalNum >= reportedDataTypeTotalNum * 1.1) {
```

Comment:
"1.1" should be a static final constant and have a descriptive name

Reply:
Fixed~